### ISSUE TITLE
XplatGenerateReleaseNotes Error: Cannot read property 'map' of undefined

### ISSUE THREAD

**Issue by gregpakes** [rfennell/vNextBuild#122]
### VSTS Extension you are using
XPlatGenerateReleaseNotes

### Where are you running it?
TFS 2017 U 1

### Version of Extension/Task
1.1.5

### Expected behavior and actual behavior

Getting the following error.  I presume there is a missing dependency?

```
2017-05-16T06:09:13.2203408Z Getting the workitems associated between release 702 and release 621
2017-05-16T06:09:13.3607411Z ##[debug]task result: Failed
2017-05-16T06:09:13.3607411Z ##[debug]Processing: ##vso[task.issue type=error;]Unhandled: Cannot read property 'map' of undefined
2017-05-16T06:09:13.3607411Z ##[error]Unhandled: Cannot read property 'map' of undefined
```

**Comment by gregpakes** [comment]
I believe this is due to the workitems variable being null.  Writing a fix now.

**Comment by rfennell** [comment]
Released in 1.2.2
